kibana/templates/deployment.yaml

```diff
@@ -64,6 +64,8 @@ spec:
         env:
           - name: ELASTICSEARCH_URL
             value: "{{ .Values.elasticsearchURL }}"
+          - name: ELASTICSEARCH_HOSTS
+            value: "{{ .Values.elasticsearchURL }}"
```

Comment:
As this is supported in 6.6.0 and elasticsearchURL won't be supported in 7.x I think it makes the most sense to add a new `elasticsearchHosts` variable which is commented out by default. And updating both environment variables to only be added if they have been defined. 

Once 7.x is the stable release we can switch them around. So that would be making `elasticsearchHosts` the default with `elasticsearchURL` being commented out so that the chart would still work with releases before 6.6.0. 

This helps to keep things in-sync with Kibana. If Kibana 7.x doesn't let you set elasticsearchURL then we shouldn't either and could cause some confusion.

Reply:
That sounds reasonable to me, pushed in https://github.com/elastic/helm-charts/pull/67/commits/e2ba0a28747ea8f46397cd14781b9c106dc439c3.